### PR TITLE
Issue #68 Optimize test

### PR DESCRIPTION
**Pull request by ethanstrominger**
Changes:
- added node_modules to .gitignore   - surprised it was not there
- added `gem 'spring-commands-rspec'`
- created bin/rspec following instructions at https://github.com/jonleighton/spring-commands-rspec/
- changed Installations.md based on findings during setup
and new command for testing

Co-authored-by: Riccardo Giomi <contact@riccardogiomi.com>

#### Fixes issue #68 

- [X] Tests all green tests and all builds passes :green_heart:
- [X] PR well-named, issue linked :paperclip: and no merge conflicts.
- [X] Request review from other contributors

### CODE REVIEW

File: Installations.md [hunk 1]
```diff
@@ -86,13 +90,13 @@ If you need to update rails, you can run `bundle update rails`.  If you run into
 
 ### Step 3: Set up the database
 
-* We using the database used in WebsiteOne. Ask for a copy of `config/database.yml`the [websiteone slack channel](https://agileventures.slack.com/archives/C029E8G80).
+rails db:create
```
Comment (aonomike) [inline Installations.md:93]: Hi @ethanstrominger, big thank you for updating these database set up instructions in the readme. We have however created an #70 to have this better clarified. Maybe we can update that from there?

**Comment by aonomike** [comment]
Hi @ethanstrominger @riccardo-giomi  , amazing work here!! Do you want to point the PR to the main develop branch the other PR doesnt block it? It may need to be used by other branches as well. Thanks :)

**Comment by aonomike** [comment]
Also maybe we can make the change to make the pass on CI as well?

**Comment by ethanstrominger** [comment]
Using a different branch

**Pull request closed without merging**